modules/update-upgrading-cli.adoc

```diff
@@ -56,7 +56,7 @@ $ oc get clusterversion -o json|jq ".items[0].spec"
 +
 [IMPORTANT]
 ====
-For production clusters, you must subscribe to a `stable-*` channel.
+For production clusters, you must subscribe to a `stable-\*` or `fast-*` channel.
 ====
 
 . View the available updates and note the version number of the update that
```

Comment:
And a few lines down, there are some old 4.1 references that we might want to freshen up.

Reply:
Good point; because that'll require some manual backports since the versions should probably match the doc version (e.g., 4.3, 4.4, etc.), I'll follow-up those changes in a separate PR.